apps/api/src/domain/models/song/commands/add-lyrics-for-song/add-lyrics-for-song.command.integration.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { AggregateType, FluxStandardAction } from '@coscrad/api-interfaces';
+import { CommandHandlerService } from '@coscrad/commands';
+import { INestApplication } from '@nestjs/common';
+import setUpIntegrationTest from '../../../../../app/controllers/__tests__/setUpIntegrationTest';
+import { IIdManager } from '../../../../../domain/interfaces/id-manager.interface';
+import { DeluxeInMemoryStore } from '../../../../../domain/types/DeluxeInMemoryStore';
+import TestRepositoryProvider from '../../../../../persistence/repositories/__tests__/TestRepositoryProvider';
+import generateDatabaseNameForTestSuite from '../../../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import { DTO } from '../../../../../types/DTO';
+import { assertCommandSuccess } from '../../../__tests__/command-helpers/assert-command-success';
+import { dummySystemUserId } from '../../../__tests__/utilities/dummySystemUserId';
+
+const commandType = 'ADD_LYRICS_FOR_SONG';
+
+validPayload: AddLyricsForSong = {
+    aggregateCompositeIdentifier: 
```

Comment:
Look at the command to see what properties to put here. You need
- `aggregateCompositeIdentifier`
- `lyrics`
- `languageCode`